CMakeLists.txt

```diff
@@ -25,7 +25,8 @@ include(AwsFeatureTests)
 include(AwsSanitizers)
 include(AwsThreadAffinity)
 include(AwsThreadName)
-include(AwsSIMD)
+#included in AwsFeatureTests.cmake
+#include(AwsSIMD)
```

Comment:
i think taking them out is worse, because you're going to want to know that it moved one day and you aren't going to know to look at git history for something you don't even know was there in the first place. But it's your maintenance burden, happy to remove

Reply:
added some comments where i think it made sense and deleted the rest.